tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -987,6 +988,7 @@ def forward_xp(self, inputs, xp):
     ((1, 1, 2), -4)
 ])
 def test_expand_dims_invalid(xp, shape, axis):
+    warnings.simplefileter('error', DeprecationWarning)
```

Comment:
- Could you surround the test code by `warnings.catch_warnings`?
- Do you think whether the test should pass if `chainerx.expand_dims` warns instead of it raises.

Reply:
Fix the 1st one.

For the 2nd:
- It depends on how much chainerx should be compatible with numpy and I don't mind much about it for now
- I like it to raise more since it may have potential bug in the warned argument